package.json

```diff
@@ -119,6 +120,8 @@
     "is-ipfs": "^0.3.2",
     "is-stream": "^1.1.0",
     "joi": "^13.1.2",
+    "joi-browser": "^13.0.1",
+    "joi-ipfs-config": "^1.0.2",
```

Comment:
@alanshaw are you proposing that we need to update an external module every time we want to tinker with the config? Sounds like a bit of too much overhead.

Reply:
I'll move it into js-ipfs